homeassistant/helpers/intent.py

```diff
@@ -252,13 +265,47 @@ def __init__(
         self.text_input = text_input
         self.context = context
         self.language = language
+        self.category = category
 
     @callback
     def create_response(self) -> IntentResponse:
         """Create a response."""
         return IntentResponse(self, language=self.language)
 
 
+class IntentResponseState(Enum):
```

Comment:
Should we just make this a boolean or do you expect future states like `processing` to be added ?

Reply:
I think a boolean would be fine. I'd expect processing to be communicated in a different message.